eq_cir_proxy_service/services/validators/request_validator.py

```diff
@@ -0,0 +1,27 @@
+"""This module contains the request validators for the EQ CIR Proxy Service."""
+
+from __future__ import annotations
+
+import semver
+from fastapi import HTTPException, status
+
+from eq_cir_proxy_service.config.logging_config import logging
+from eq_cir_proxy_service.exception import exception_messages
+
+logger = logging.getLogger(__name__)
+
+
+def validate_version(version: str) -> None:
+    """Checks if the version is a valid semver.
+
+    Parameters:
+    - version: The version to validate.
+    """
+    try:
+        semver.VersionInfo.parse(version)
```

Comment:
Should we use the specific classmethod in here [semver.VersionInfo.isvalid()](https://python-semver.readthedocs.io/en/2.9.1/api.html#semver.VersionInfo.isvalid) for clarity.